lib/job-simulation.rb

```diff
@@ -5,11 +5,34 @@ class JobSimulation
   attr_reader :workers, :waiting, :roll
 
   def initialize (jobs_available, job_seekers)
-
+    @jobs_available = jobs_available
+    @job_seekers = job_seekers
```

Comment:
What is the purpose of saving these passed variables into new instance variables with the same name? If you aren't going to use them in another method (which you shouldn't need to do) then you don't need to do this.

Instead you could not do lines 8 and 9, and then remove the @ in front of the vars below, and you'd still get the same result.